### PR TITLE
BUG: plotting layout patch

### PR DESCRIPTION
**Pull request by lyashevska**
This fixes a layout problem.

### CODE REVIEW

**Comment by jreback** [comment]
hmm can you show the before / after. also please add a whatsnew note in plotting / bug fixes / 1.1.

**Comment by lyashevska** [comment]
before
after

**Comment by jreback** [comment]
kk great, pls add a whatsnew (e.g. when a user would care about this) and ping on green.

File: doc/source/whatsnew/v1.1.0.rst [hunk 1]
```diff
@@ -1006,6 +1006,7 @@ Plotting
 - Bug in :meth:`DataFrame.hist` where the order of ``column`` argument was ignored (:issue:`29235`)
 - Bug in :meth:`DataFrame.plot.scatter` that when adding multiple plots with different ``cmap``, colorbars alway use the first ``cmap`` (:issue:`33389`)
 - Bug in :meth:`DataFrame.plot.scatter` was adding a colorbar to the plot even if the argument `c` was assigned to a column containing color names (:issue:`34316`)
+- Bug in :meth:`Series.bootstrap_plot` was cluttered axes and overlapping labels (:issue:`34905`)
```
Comment (jreback) [inline doc/source/whatsnew/v1.1.0.rst:1009]: this reference doesn't exist, i think you need pandas.plotting.bootstrap_plot, ping when green.

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/whatsnew/v1.1.0.rst (hunk 1 above)
Comment (lyashevska) [reply doc/source/whatsnew/v1.1.0.rst]: I think it should be solved now, could you please have a look?

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @lyashevska

**Pull request merged**